ingenialink/utils/mcb.py

```diff
@@ -134,8 +135,10 @@ def read_mcb_data(cls, expected_address, frame):
         recv_add, _, cmd, data = cls.read_mcb_frame(frame)
 
         if cmd != MCB_CMD_ACK:
-            err = frame[cls.DATA_START_BYTE : cls.DATA_END_BYTE].hex()
-            raise ILNACKError(f"Communications error (NACK -> {err[::-1]})")
+            err = frame[cls.DATA_START_BYTE : cls.DATA_END_BYTE]
```

Comment:
Maybe, you can remove this line because the data is already the error.

Reply:
Done.